tingbot/platform_specific/sdl_wrapper.py

```diff
@@ -49,6 +49,7 @@ def click(self, xy, action):
             self.surface.fill(self.color)
             if button_callback:
                 button_callback(self.number, 'up')
+        screen.needs_update = True
 
```

Comment:
If a button doesn't actually change the screen, then it is not animated in the simulator. This only shows up in certain use cases